docs/tables/hypothesis_profile.md

```diff
@@ -1,12 +1,12 @@
 # Table: hypothesis_profile
 
-If you [authenticate](../index.md), this table reports your Hypothesis username, display name, authority, and groups.
+If you [authenticate](https://hub.steampipe.io/plugins/turbot/hypothesis#credentials), this table reports your Hypothesis username, display name, authority, and groups.
```

Comment:
@judell I updated this link since I believe it's better to link directly to the Hub page in terms of how the table docs are built. There may be a way to have relative links with anchors, but I'm not sure on the syntax, and it's difficult to test since we can't test plugin doc changes easily without pushing new production tags.

Reply:
Makes sense. This is yet more fodder for thinking through how to write a plugin that:

- you build and run locally

- then lands in a non-turbot repo for others to build and locally

- then makes its way to the hub (and eventually steampipe cloud)